packages/m2m-gateway/test/api/tenants/getTenants.test.ts

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
```

Comment:
Let's move this to the tests eslintrc rules to ignore so we don't have to add it to each file https://github.com/pagopa/interop-be-monorepo/pull/1831/files#diff-5d1e9ba5db2a8bb4e930b9a3cf673c0179641f6e38ea42dbfe637975cc81d6f3R8